lib/twingly/public_suffix_list.rb

```diff
@@ -1,3 +1,4 @@
+require "addressable/idna"
```

Comment:
Thanks, didn't check the full file before so I didn't spot the call to to_ascii.